src/styles/components/pod-configuration-tab.less

```diff
@@ -0,0 +1,12 @@
+
+.pod-config-container-group {
+    margin-left: 8px;
+    border-left: solid 2px #9351E5;
+    padding-left: 12px;
+}
+
+.pod-config-network-group {
```

Comment:
Perhaps we should break this into two different classes:

* `pod-config-container-group` - That contains the general rules (margin, padding, border style)
* `pod-config-container-group-network` - That adds the network color
* `pod-config-container-group-container ` - That adds the container color